.travis.yml

```diff
@@ -5,7 +5,11 @@ matrix:
     - python: 3.7
       dist: xenial
       sudo: true
+before_install:
+  - pip install --upgrade pip
+  - pip --version
 install:
+  - pip install multiprocess
```

Comment:
We should figure out why we need to install multiprocess explicitly.

Reply:
I'm not satisfied with this either. I think I know how to fix this.